### ISSUE TITLE
Support hotfix releases for Aerie and Aerie UI

### ISSUE THREAD

**Issue by camargo** [NASA-AMMOS/aerie#630]
Sometimes we just need to release a hotfix with one (or a couple) commits to quickly fix blocking issues. We need to come up with a simple strategy to do this in our CI pipeline.

**Comment by parkerabercrombie** [comment]
In past projects we've addressed this by maintaining a release branch for each delivery (or create a branch based on your release tag when you need to hotfix).  So in the case of the 1.0.2 release, you would create branch `1.0.2` from the release tag, commit the fix for the hotfix ticket to the `1.0.2` branch, and deliver a new point release based on that branch. And then you need to cherry-pick or rebase those change onto `develop` so it gets into future releases.

It does add some work to maintain release branches this way, but it does greatly reduce the risk of accidentally introducing new issues into a point release.

**Comment by skovati** [comment]
Similar to @parkerabercrombie's idea above, a process I've used before is to have a `develop` branch and a `release` branch. `develop` exists as the main iterative branch, i.e. new features are merged into `develop` and once a release is ready (planned point release or ad-hoc hotfix), a PR is made into `release` which will kick off release automation. For hotfixes, this PR against `release` would just be a few cherry-picked commits.

**Comment by camargo** [comment]
We support hotfix releases now and have done a couple. See:

1. https://github.com/NASA-AMMOS/aerie/releases/tag/v1.6.2
2. https://github.com/NASA-AMMOS/aerie/releases/tag/v1.6.1
